### PR TITLE
add k8s-infra-staging-scl-image-builder group

### PR DESCRIPTION
**Pull request by moshloop**

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -742,6 +742,20 @@ groups:
       - thockin@google.com
       - bartek@smykla.com
 
+  - email-id: k8s-infra-staging-scl-image-builder@kubernetes.io
+    name: k8s-infra-staging-scl-image-builder
+    description: |-
+      ACL for Sig Cluster Lifecycle Image Builder project
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
```
Comment (ameukam) [inline groups/groups.yaml]: Email delivery is not used for this group. You can drop this line.

Comment (moshloop) [reply groups/groups.yaml]: Fixed

File: groups/groups.yaml [hunk 2]
```diff
@@ -714,6 +714,18 @@ groups:
       - fabrizio.pandini@gmail.com
       - neolit123@gmail.com
 
+  - name: k8s-infra-staging-scl-image-builder
```
Comment (ameukam) [inline groups/groups.yaml]:
```suggestion
  - email-id: k8s-infra-staging-scl-image-builder@kubernetes.io
  	name: k8s-infra-staging-scl-image-builder
```

**Comment by thockin** [comment]
Ping @justinsb @timothysc

File: groups/groups.yaml [hunk 3]
```diff
@@ -714,6 +714,18 @@ groups:
...
+    description: |-
+      ACL for Sig Cluster Lifecycle Image Builder project
```
Comment (neolit123) [inline groups/groups.yaml]: Sig -> SIG

File: groups/groups.yaml [hunk 4]
```diff
@@ -714,6 +714,18 @@ groups:
...
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - moshe.immerman@gmail.com
+      - justinsb@google.com
+      - timothysc@gmail.com
+      - trhoden@gmail.com
+      - luoh@vmware.com
```
Comment (justinsb) [inline groups/groups.yaml]: Cross checked these people against https://github.com/kubernetes-sigs/image-builder/blob/master/OWNERS

:+1:

**Comment by justinsb** [comment]
+1 from me; these people are OWNERs of the image-builder subproject.

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**